python/valis/routes/target.py

```diff
@@ -195,6 +196,26 @@ async def get_catalogs(self, sdss_id: int = Path(title="The sdss_id of the targe
 
         return response_data
 
+    @router.get('/parents/{parent_catalog}/{sdss_id}',
+                dependencies=[Depends(get_pw_db)],
+                response_model=ParentCatalogModel,
+                summary='Retrieve parent catalog information for a taget by sdss_id')
+    async def get_parents(self,
+                          parent_catalog: str = Path(title='The parent catalog to search',
+                                                     example='gaia_dr3_source'),
+                          sdss_id = Path(title='The sdss_id of the target to get',
+                                         example=129055990)):
```

Comment:
Can you make these Annotated?   I forgot to mention this in the last PR.  FastAPI is now recommending that method for declaring additional metadata and default values for path and query parameters.  See https://fastapi.tiangolo.com/tutorial/path-params-numeric-validations/.   Here is a valis example, https://github.com/sdss/valis/blob/b30d03db351bf564a8303a6ddb7f6e488fb993b6/python/valis/routes/target.py#L172

Reply:
Ah yes, I read about that and propmtly forgot it. BTW, it seems that `title` doesn't show up in Swagger and one needs to use `description`.

Reply:
oh good to know!